### PR TITLE
Mb odd

### PR DESCRIPTION
**Pull request by mattbruton**
Created Odd class, created unit tests that focus on what was not inherited from Command class.

### CODE REVIEW

**Comment by katerebekah** [comment]
Don't forget to tag me next time :angry: :wink: :+1:

**Comment by ZachParris** [comment]
Awesome, Matt! Super clear code 👍  Should have Integer and Even up shortly!

File: Mathmagician.Tests/OddTests.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            {
+                if(number % 2 == 0)
+                {
+                    hasEvenNumbers = true;
+                }
+            }
+
+            Assert.IsFalse(hasEvenNumbers);
+        }
+    }
```
Comment (jcockhren) [inline Mathmagician.Tests/OddTests.cs:54]: @mattbruton I think this test can be re-written.

**Comment by mattbruton** [comment]
Will rewrite the test as soon as I wrap up the task I'm working on, @jcockhren .

**Comment by jcockhren** [comment]
@mattbruton [Use the All method](https://msdn.microsoft.com/en-us/library/bb548541%28v=vs.110%29.aspx) or [Any](https://msdn.microsoft.com/en-us/library/system.linq.enumerable.any%28v=vs.110%29.aspx)

**Comment by katerebekah** [comment]
Since we haven't done the Any method yet (or any of the functional style methods like that), let one of us know if you run into trouble with it

**Pull request merged**